### DISCUSSION TITLE
Supported device not supported on home assistant

### DISCUSSION THREAD

**Discussion by nonyhaha** [Koenkk/zigbee2mqtt#9656] [Devices]
Hello, 
I just bought 2 Ikea bulbs - new model, https://www.zigbee2mqtt.io/devices/LED2003G10.html . 
I paired it to my installation of home assistant but it appears as not recognised, although the device is in the ikea.js file on the repository. 
I am running both stable and edge addons, but the bulb appears as unsupported on both integrations. 
I am not sure how to check if the latest ikea.js is on any of my installations of zigbee2mqtt running on home assistant supervised. 
What should I do in this situation?

**Comment by nonyhaha** [comment]
My problem was an old version of zigbee2mqtt addon, that is missing current updates. I had to switch to the new repository link.

**Reply by GrohmS2977** [reply]
Can you tell me how to update the zigbee2mqtt addon in HA? I have the same problem with a new heating thermostat -> not supported. I run version 1.18.1-1 but think i need a newer version. Automatic updates are on but nothing happens.

**Reply by nonyhaha** [reply]
> Can you tell me how to update the zigbee2mqtt addon in HA? I have the same problem with a new heating thermostat -> not supported. I run version 1.18.1-1 but think i need a newer version. Automatic updates are on but nothing happens.

Hi. It is pretty easy. You go to supervisor section, addon store, 3 dots -> repositories, add there the new repositoruly link from zigbee2mqtt instructions on their website, then aearch in the addon store for zigbee2mqtt an you should find the newer version. Good luck!

**Reply by GrohmS2977** [reply]
thank you for your fast response! Since i already have installed the Zigbee2mqtt addon, will it then install a second one or overwrite the old one? Would cause some panic on my side if then there are 2 addons :-D

![grafik](https://user-images.githubusercontent.com/97183711/148255347-96be9396-37b6-49b0-a66d-aad2af9b8c71.png)

**Reply by nonyhaha** [reply]
There will be a new addon installed. Make a backup of the old one just to be sure. Depending on the firmware on your zigbee stick, it might have issues. Repairing might be required.

**Reply by GrohmS2977** [reply]
ok thank you, i will give it a try! (Y)

**Reply by GrohmS2977** [reply]
Update: I tried it as described below and it works fine so far! Thanks for your support!

https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/52#issuecomment-792873888

**Reply by p0ske** [reply]
Thank you so much, resolved my issue as well!
